### PR TITLE
TAVC-1603 Added endpoint to get registration details

### PR DESCRIPTION
**Pull request by CoolName1993**
Also refactored code to use Microservice app config.

### CODE REVIEW

File: conf/app.routes [hunk 1]
```diff
@@ -10,5 +10,7 @@ GET        /averaged-annual-turnover/check-averaged-annual-turnover/proposed-inv
 
 POST       /advanced-assurance/:tavcReferenceId/submit                                                                                                                                                                                                                             controllers.SubmissionController.submitAA(tavcReferenceId: String)
 
+GET        /registration/get-registration-details/safeid/:safeID                                                                                                                                                                                                                   controllers.RegistrationDetailsController.getRegistrationDetails(safeID: String)
```
Comment (HarryGull) [inline conf/app.routes]: A good point recently that the route doesn't need to be /registration/get-registration-details/safeid/:safeID  as it implicit in the GET verb. 
so /registration/registration-details/safeid/:safeID  means we could reuse the same route if ever needed a POST/PUT etc.

File: conf/application.conf [hunk 2]
```diff
@@ -172,7 +172,17 @@ microservice {
...
+            host=localhost
+            port=9639
+        }
+
     }
 }
 
-environment = "ISIT"
\ No newline at end of file
+environment = "ISIT"
```
Comment (HarryGull) [inline conf/application.conf]: Just a query - didn't you say this was required to be lower case?

File: conf/application.conf [hunk 3]
```diff
@@ -172,7 +172,17 @@ microservice {
...
+get-registration-details {
+    url = "/tax-assured-venture-capital/registration/details"
```
Comment (HarryGull) [inline conf/application.conf]: Is this correct?
Shouldn't this match the DES URL - somethign like taxpayers/XX/XX

Comment (CoolName1993) [inline conf/application.conf]: The API url is /registration/details?safeid={safeId}, so I can get rid of the tax-assured-venture-capital part but I will need to update the routes files in the stub

File: conf/application.conf (hunk 2 above)
Comment (CoolName1993) [inline conf/application.conf]: Yeah, I'll update this now

**Comment by HarryGull** [comment]
Look good

**Pull request merged**